### PR TITLE
Web Page Clone of The Next Web page

### PR DESCRIPTION
**Pull request by abdelp**
Responsive design of The Nex Web page

Based on the Odin Project Requirements specified here: [https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design](https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-responsive-design)

- use of flexbox, CSS Grid for the layout
- use of media queries
- HTML semantic tags
- use of gradients

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1164 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Hello World</title>
+  <link rel="stylesheet" href="assets/stylesheets/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Poppins:300,400,600,700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="assets/stylesheets/style.css">
+  <link rel="stylesheet" href="assets/stylesheets/big-size.css">
+  <link rel="stylesheet" href="assets/stylesheets/mid-size.css">
+  <link rel="stylesheet" href="assets/stylesheets/little-size.css">
+  <!-- <link rel="stylesheet" href="assets/stylesheets/styles-mediaquery-reporter.css"> -->
```
Comment (pelzolga123) [inline index.html]: Commented line should be deleted.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
Just a few corrections needed before project can be approved:
 * [ ] Navbar looks a little messed up in medium size version, make it look more like original. Your version:
image

 Original:
 image

* [ ] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by abdelp** [comment]
removed commented code
fixed navbar display for medium size devices

**Comment by pelzolga123** [comment]
## Great  work so far
But you still need to fix next:
* [ ] All stickler errors should be fixed.
* [ ] Medium version looks now like small one. For small version max-width should be around 700, right now it's around 800.
* [ ] Navbar still looks not like original one, second row should not be empty:
image

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

**Comment by andresporras3423** [comment]
* small version max-width should be around 737
* the second row is not appearing anymore in the small version
* linter issues fixed

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
## Hi team!
### Nice job! Project looks! :+1: 

## Status [Approved]


You have resolved all the issues raised by the previous TSE




#### Go ahead and merge to master.
#### Goodluck in your next project Happy coding

**Review by Anna-Myzukina — APPROVED** [review]

**Comment by abdelp** [comment]
Thank you very much!

**Pull request merged**